deleted:    README.md

MrCodemex opened the pull request:
new file:   index.html
	new file:   style.css

MrCodemex committed:
deleted:    README.md
	new file:   index.html
	new file:   style.css

VanillaWulf commented on index.html:
```diff
@@ -0,0 +1,173 @@
+    </main>
+
+        <section class="main-page-subscribe">
+            <h2 class="main-page-categories__h2-invisible">Subscribe</h2>
```
нельзя делать текстовые названия h2 p

VanillaWulf commented on index.html:
```diff
@@ -0,0 +1,173 @@
+                    <li class="right-footer"><p>© 2021  Brand  All Rights Reserved.</p>
+                    </li>
+                    <li class="left-footer">
+                        <img src="images/fb.svg" alt="">
```
это список ссылок

VanillaWulf commented on index.html:
```diff
@@ -0,0 +1,173 @@
+        <footer>
+            <div class="footer-container">
+                <ul class="footer-container__list">
+                    <li class="right-footer"><p>© 2021  Brand  All Rights Reserved.</p>
```
разная стуртура у элементом

MrCodemex committed:
modified:   index.html
deleted:    normalize.css
new file:   registration.html
new file:   reset.css
modified:   style.css
